sympy/series/formal.py

```diff
@@ -1210,6 +1219,77 @@ def __rmul__(self, other):
         return self.__mul__(other)
 
 
+class FiniteFormalPowerSeries(FormalPowerSeries):
+    """Represents Finite Formal Power Series of a function.
```

Comment:
The coefficient sequence of the objects of the class are finite, since they are lists calculated on the basis of the order `n` specified in the related function.